### ISSUE TITLE
Support GLPK 5.0.

### ISSUE THREAD

**Issue by Viech** [biosustain/swiglpk#41]
Installing `swiglpk 4.56.1` on top of `GLPK 5.0` works but `import swiglpk` fails:
```
ImportError: /usr/lib/python3.9/site-packages/swiglpk/_swiglpk.cpython-39-x86_64-linux-gnu.so: undefined symbol: glp_netgen_prob
```
This [release mail](https://lists.gnu.org/archive/html/info-gnu/2020-12/msg00007.html) explains the relevant change.

**Comment by Viech** [comment]
I noticed that your wheel release `swiglpk-4.65.1-cp27-cp27mu-manylinux1_x86_64.whl` is not affected; it works fine with GLPK 5.0. I could not test a Python 3 wheel since none are available for Python 3.9 yet but it stands to reason that only swiglpk installed via `setup.py install` would be affected by the `ImportError` above.

Note that I haven't tested if an existing installation via `setup.py install` would continue to work when GLPK is upgraded afterwards; for me GLPK was already updated to 5.0 when I installed swiglpk.

**Comment by cdiener** [comment]
I can reproduce that. This should only affect Python versions>3.8 since those provide no wheels and are thus built on the host system which will produce the version mismatch.

**Comment by Viech** [comment]
@phantomas1234, may I ask if you or someone else at @biosustain are planning to catch up with the current and with future GLPK releases? (swiglpk seems to be the last lightweight/low-level interface that's still around.)

**Comment by Viech** [comment]
I can confirm that `master` is not affected anymore. For Arch Linux, I've added an additional [git package](https://aur.archlinux.org/packages/python-swiglpk-git/) to the AUR that can be used until a release is made.

**Comment by carrascomj** [comment]
Closed by #48 (working on linux, macos and windows, finally!): https://pypi.org/project/swiglpk/
